lib/board.rb

```diff
@@ -64,13 +64,17 @@ def place(ship, coordinates)
 	def render(boolean = false)
 		display = "  "
 		@size.times do |width|
-			display += "#{width + 1} "
+			if width < 8
```

Comment:
I like it, especially since we have our board limited in size this should look great.